src/components/learner-credit-management/search/CatalogSearchResults.jsx

```diff
@@ -118,6 +126,7 @@ BaseCatalogSearchResults.defaultProps = {
 };
 
 BaseCatalogSearchResults.propTypes = {
+  enterpriseSlug: PropTypes.string.isRequired,
```

Comment:
Given the only reason `enterpriseSlug` is passed as a prop to this component is to pass it along to the child `CourseCard` components, it may be a good idea to just connect `CourseCard` itself to the Redux store to avoid prop drilling. This would simplify `CatalogSearchResults` since you wouldn't need to create a wrapper around the `CourseCard` component just to get the `enterpriseSlug` prop.